reconcile/utils/mr/cluster_machine_pool_updates.py

```diff
@@ -0,0 +1,74 @@
+from io import StringIO
+from typing import Any
+
+from ruamel.yaml import YAML
+
+from reconcile.change_owners.decision import DecisionCommand
+from reconcile.utils.gitlab_api import GitLabApi
+from reconcile.utils.mr.base import MergeRequestBase
+
+yaml = YAML()
+yaml.explicit_start = True
+# Lets prevent line wraps
+yaml.width = 4096
```

Comment:
can reuse `create_ruamel_instance` for duplicate config

Reply:
done